pymc/model.py

```diff
@@ -2035,10 +2035,43 @@ def Deterministic(name, var, model=None, dims=None):
 def Potential(name, var, model=None):
     """Add an arbitrary factor potential to the model likelihood
 
+      Potential function is used to add arbitrary factors i.e. constraints that we define to the likelihood function of a model to adjust the probablity density of the model. 
+
+    .. code:: python
+
+        with pm.Model() as my_model:
+          x = pm.Normal('x', mu=0, tau=1)
+          y = pm.Normal('y', mu=x, tau=1)
+
+          constraint = x + y < 1
+          potential = pm.Potential('constraint', constraint, my_model)
```

Comment:
I wouldn't show the `model` argument being used. Most times it's not needed as users add a Potential inside a model context.